bedrock/firefox/redirects.py

```diff
@@ -604,7 +604,7 @@ def firefox_channel(*args, **kwargs):
     redirect(r'^firefox/firefox\.html$', 'firefox.new'),
 
     # issue 6979
-    redirect(r'^firefoxfightsforyou/?', 'firefox.fights-for-you'),
+    redirect(r'^firefoxfightsforyou/?', 'https://www.mozilla.org/firefox/'),
```

Comment:
```suggestion
    redirect(r'^firefoxfightsforyou/?', 'firefox'),
```

Note that the above is also just a vanity redirect. You'll also need to add a redirect for the actual page URL, which is `/firefox/fights-for-you/`